src/Hedgehog/Size.fs

```diff
@@ -0,0 +1,53 @@
+namespace Hedgehog
+
+/// Tests are parameterized by the `Size` of the randomly-generated data,
+/// the meaning of which depends on the particular generator used.
+[<Struct>]
+type Size = private Size of struct(int * int)
+
+module Size =
+
+    let create (current : int) (maximum : int) : Size =
+        Size (current, int maximum)
+
+    let constant (n : int) : Size =
+        create n n
+
+    let current (size : Size) : int =
+        let (Size (current, _)) = size
+        int current
```

Comment:
The `int` cast is a hold over from when I had unit of measures for both members of the tuple, these will be removed. I do think it's worth having names for all of the parameters, otherwise the compiler generates useless names like `_arg1` that don't help when using tools to browse the code.